lib/msf/core/post/windows/services.rb

```diff
@@ -528,7 +528,7 @@ def service_restart(name, start_type=START_TYPE_AUTO, server=nil)
   # Parses out a SERVICE_STATUS struct from the
   # lpServiceStatus out parameter
   #
-  # @param (lpServiceStatus)
+  # @param (lpServiceStatus) lpServiceStatus latest status of calling service
```

Comment:
This was wrong before. Parens indicate a default. This should be something like:
```
# @param lpServiceStatus [String] latest status of calling service
```